### DISCUSSION TITLE
Data sources: Biological

### DISCUSSION THREAD

**Discussion by melissamonk-NOAA** [pfmc-assessments/CA_quillback_rockfish_2025#5] [General]
List of possible data sources to inform biology

1. Otoliths and lengths
- [x]    a. Recent CDFW
- [x]    b. NMFS Cooperative research
- [x]    c. CCFRP
- [x]    d. Abrams thesis
- [x]    e. Commercial
- [x]    f. WCGBTS
- [x]    g. IPHC FISS survey
2. Length only 
- [x]    a. Miller and Gotshall 1965
- [x]    b. Miller and Geibel 
- [x]    c. Wade vanBuskirk lengths
- [x]    d. Recreational (See recreational data)
- [x]    e. CDFW ROV
- [x]    f. Diana Baetschers dissertation data of quillback YOY lengths
- [x]    g. Triennial
3. Weight 
- [x]   ~a. Commercial. UPDATE: No commercial weights~
- [x]   b. Recreational - Weights available for about half the entries. This is the pimary source 
- [x]   c. WCGBTS
4. Maturity
- [x]    a. NMFS SWFSC 2024 collection
- [x]    ~b. PacFIN has agency maturity code. Need to confirm these are or are not suitable. UPDATE: These are not suitable~
6. Fecundity
- [x]     a. NMFS SWFSC 2024 collection
- [x]     b. Dick et al. 2017 estimates

**Comment by brianlangseth-NOAA** [comment]
Historical data should be the same as that used in the 2021 assessment

**Comment by melissamonk-NOAA** [comment]
Check that DWV and MRFSS lengths aren't double counted

**Reply by brianlangseth-NOAA** [reply]
Im going to move this to an issue. #19

**Comment by brianlangseth-NOAA** [comment]
In a CDFW report [Agenda I.4.b Supplemental CDFW Report 1](https://www.pcouncil.org/documents/2024/09/i-4-b-supplemental-cdfw-report-1-cdfw-report-on-2025-and-2027-stock-assessment-plan-and-schedule.pdf/)at the September 2024 council meeting, CDFW outlined the number of otoliths available though a couple of sampling programs. 

![image](https://github.com/user-attachments/assets/b87a19c8-f8f8-47d8-aaa5-91dfd5bc284c)

Of note, the FISS column reflects the IPHC Fishery Independent Setline Survey, which is likely a dataset we should look at

**Comment by brianlangseth-NOAA** [comment]
There is a huge fish in recfin that also has a not unrealistic (for its size) weight. Heard any stories of this thing (sampled in 2012)? Seeing as this would be an all time record for length (and four all time records for weight, which are in kg), I do not plan to keep this length, nor these weights (not that we use them).  
![image](https://github.com/user-attachments/assets/0fd96d53-8650-43e4-bf13-64dae1058dd1)

**Reply by brianlangseth-NOAA** [reply]
Melissa and I discussed and these 4 seem reasonable to remove. Also agreed the two other long fish without weights be removed

**Comment by brianlangseth-NOAA** [comment]
@melissamonk-NOAA and @JuliaCoates, MRFSS data has two different length measurements for fish: fork length (LNGTH) and total length (T_LEN). Per the discussion at pfmc-assessments/california-data#2, LGNTH was measured >= 1993, and then T_LEN was measured <= 1993. Thus we have two different measured values

![image](https://github.com/user-attachments/assets/edc64cc9-e735-444b-bbc1-dcd6e16dbc83)

As you can see in the above figure there is a set of 1997-1998 data where LNGTH wasn't measured (black dots = LNGTH with many decimals). My guess is that this might be Deb's data and is something I plan to look into.

We also need to deal with the fork length / total length issue. For quillback rockfish this should not be an issue because theoretically, the two measurements should be the same. However, it is clear there is a difference between fork and total length, and this difference is larger the bigger the fish is. Most of the time total length is greater than fork length but not always. 

![image](https://github.com/user-attachments/assets/b80aa55d-b04b-4093-a4e1-c19b79d5d1c1)

Since total length was used in the earliest years, IF total and fork were the actually the same as we would expect, the conversion being applied in the MRFSS data would generally make fish smaller in the earliest years. As there is no way to know for sure how the measurement was taken, **my suggestion is that we move forward with fork length (which is what was done in the last assessment). We can discuss this next time we meet to confirm we are in agreement**.

**Comment by brianlangseth-NOAA** [comment]
From Echeverria and Lenarz 1984, the length conversions are:

TL = -8.696 + 1.034 * FL
FL = 9.075 + 0.965 * TL

These are for lengths in mm

**Comment by brianlangseth-NOAA** [comment]
Information on IPHC FISS survey from @JuliaCoates:

The survey’s southern extent is SF Bay. Total numbers of quillback caught COASTWIDE 

![image](https://github.com/user-attachments/assets/cbda5ab6-dc55-400b-957b-db477f3d70c1)'

but among CALIFORNIA locations, the survey only extended into California in 2013, 2014, and 2017. No quillback were caught in California in 2013, 22 in 2014, and 5 in 2017. CDFW has the data for the 5 fish in 2017. Looking into getting the data in 2014.

**Reply by brianlangseth-NOAA** [reply]
Only going to have the 5 fish in 2017. Not getting other data, per email from @JuliaCoates on Jan 21, 2025

**Comment by brianlangseth-NOAA** [comment]
Compilation of the length data started in commit b0f988f

**Comment by brianlangseth-NOAA** [comment]
Commit 9ec3121 shows figure of weight and length data. Based on recreational samples, primarily, the relationship is very similar to last assessment.

**Comment by brianlangseth-NOAA** [comment]
Commit e0ec3b1 provides details of the maturity analysis. Within that commit is code showing results when Melissa Head added samples from Oregon (astoria) along with those from California, as there were originally some concerns about the uncertainty around the L50 estimate, but which were later found to be based on incorrect calculations. Regardless, when Oregon data are added, L50 increased from 28.56 to 29.30. 

This increase is small and provides evidence that there isn't a large change between Oregon and California. We will use the estimate from the California data only (28.56cm)

**Comment by melissamonk-NOAA** [comment]
Diana Baetscher's dissertation data: 201 genetically identified quillback from SMURFs in Monterey Bay.  For +/- 15 days around July 1 the average length is 3.8 cm.  There may be a second pulse of recruitment later in the year. Most of the data are from a single year, 2016.
![image](https://github.com/user-attachments/assets/cb56160e-929f-4d39-9faa-4c7f11a232ec)

**Reply by brianlangseth-NOAA** [reply]
@melissamonk-NOAA from the quillback_growth.R file, the L0 value is 3.9. Im assuming this is a proper L0, and not L1 (versus L2), in which case I would be in favor of setting age at L1 to be 0 (to go along with age at L2 to be 999 - so functionally Linf). Do you know whether the L0 in that script is indeed for age 0 fish? You support setting L1 to be length at age 0? The fact that it aligns well with Dianna's measurement is comforting

**Reply by melissamonk-NOAA** [reply]
@brianlangseth-NOAA The parameterization used is $L_{\infty} − (L_{\infty}−L_0)e^{−Kt}$ so yes, $L_0$ is the length at time zero or birth.  Yes, we could set L1 in stock synthesis to 0.

**Reply by brianlangseth-NOAA** [reply]
Thanks for confirming

Right now this is what I have used. Set L1 (which is length at the smallest age, which we set to 0) to 3.9. What this has meant practically is that the population bin size has to be set to 2 (previously it was 4 which was larger than length at age 0). We can explore what the implications are of setting length at age 0 to 4 (instead of 3.9), which would avoid setting it to 2 because of the popluation bin structure.

**Comment by brianlangseth-NOAA** [comment]
I updated our growth analysis (quillback_growth.R) with the newest dataset (QLBK_Data_Dump_updated.csv) and noticed some things. First the resulting new relationship is quite different (the old one in black (K = 0.178m Linf = 41, L0 = 3.9) and new one in red (K = 0.112, Linf = 42.9, L0 = 15.66)

![image](https://github.com/user-attachments/assets/badc9d24-2e06-48dd-bb29-51bdf1f68b51)

I was a little surprised by this given that the difference in data is not that great (see the new data in red below)

![image](https://github.com/user-attachments/assets/3d344f1d-6a44-4842-8eb6-b158f03fa062)

Turns out that our old estimates are pretty sensitive to starting conditions. If our starting condition for Linf is 40 instead of 55, then the estimate based on the old data is much more similar to the estimates using the new data with original starting conditions. Basically, there is a tradeoff with fitting the few young data points with the much more numerous older data points. With the new data, the starting conditions dont make much of a difference at all.

This shouldn't affect us too much, since we plan to estimate model paramters, but in terms of expectations, we should change them to be more reflective of a flater and older curve than what was shown at the data workshop. 

I plan to update the initial model parameters with these new estimates, not only because the data is updated, but also because the new estimates seem to be more in line with the data (regardless of if its old or new) and not sensitive to starting conditions in the estimation.

**Comment by melissamonk-NOAA** [comment]
I looked at our data collected from 24 quillback rockfish in California to estimate fecundity.  The 24 fish (48 egg count samples) span a range of lengths.  The Dick et al. meta-analysis uses 253 estimates of fecundity from quillback rockfish from the commercial fishery on the inside passage of British Columbia.  E.J.'s meta-analysis is based on total length, and the estimates here are based on fork length. I applied the bias correction and the parameters are alpha = 4.215646e-08 and beta = 4.440432 where units are millions of eggs and fork length in cm. The sample sizes are reasonable given the time frame of spawning and need to find females with stage 2 eggs.  Additional fecundity data will always be needed. 
[Beyer et al. 2015 ](https://link.springer.com/article/10.1007/s10641-014-0238-7)showed varying fecundity by latitude for chilipepper and yellowtail rockfish.
![image](https://github.com/user-attachments/assets/94489135-41c7-4cc3-93b4-4520e3ef44da)

As a check, the length weight relationship (somatic weight) between the samples from Canada and California are similar
![image](https://github.com/user-attachments/assets/43bb99cf-1692-4df5-8819-e07d285cc44a)

The plot below shows the the fecundity relationship fit to E.J.'s meta-analysis (blue), fit to the available data from Canada, re-analyzed using fork length (purple) and the estimate for California only fish in red.
![image](https://github.com/user-attachments/assets/25c3690e-7017-4798-bf79-4e74ab7d278a)
